.gitignore

```diff
@@ -56,7 +56,9 @@ docs/_build/
 
 # vscode history
 .history
+.vscode/
 
 # Virtual env
 myenv/
-pyvenv.cfg
\ No newline at end of file
+pyvenv.cfg
+secretKey.json
```

Comment:
See my comment below regarding the key name, if it's different for everyone, then I think it's better not to add it to the .gitignore.